common/xrdp_client_info.h

```diff
@@ -24,15 +24,42 @@
 #if !defined(XRDP_CLIENT_INFO_H)
 #define XRDP_CLIENT_INFO_H
 
+/*
+ * 2.2.1.3.6.1 Monitor Definition (TS_MONITOR_DEF)
+ * 2.2.1.3.9.1 Monitor Attributes (TS_MONITOR_ATTRIBUTES)
+ * 2.2.2.2.1 DISPLAYCONTROL_MONITOR_LAYOUT
+ */
 struct monitor_info
 {
+    /* From 2.2.1.3.6.1 Monitor Definition (TS_MONITOR_DEF) */
     int left;
     int top;
     int right;
     int bottom;
+    int flags;
+
+    /* From 2.2.2.2.1 DISPLAYCONTROL_MONITOR_LAYOUT */
+    int width;
+    int height;
+    int physical_width;
+    int physical_height;
+    int orientation;
+    int desktop_scale_factor;
+    int device_scale_factor;
+
+    /* Derived setting */
     int is_primary;
 };
 
+struct display_size_description
+{
+    int monitorCount; /* number of monitors detected (max = 16) */
+    struct monitor_info minfo[CLIENT_MONITOR_DATA_MAXIMUM_MONITORS]; /* client monitor data */
+    struct monitor_info minfo_wm[CLIENT_MONITOR_DATA_MAXIMUM_MONITORS]; /* client monitor data, non-negative values */
+    int session_width;
+    int session_height;
+};
+
```

Comment:
I know. I didn't want to change xrdp_client_info since that will incur a revision of the header file and also affect xorgxrdp. I want to make the change to unify these two, but not in this PR... I want to do it for a PR that's focused on unifying these two data structures and leave the ones that change xorgxrdp more focused and linked. Make sense?

Reply:
Yes that's fine- you can raise a PR now and mark it as draft if you like to keep the book-keeping simple.

Reply:
I decided to actually just do it now. It's not as terrible as I thought.